<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionWindow</class>
 <widget class="QWidget" name="OptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Настройки</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QRadioButton" name="rb1">
            <property name="text">
             <string>Лёгкий</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb2">
            <property name="text">
             <string>Средний</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Сложный</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb4">
            <property name="text">
             <string>Настроить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QGridLayout" name="costum_3" rowstretch="0,0,0">
          <item row="2" column="3">
           <widget class="QLCDNumber" name="lcd3"/>
          </item>
          <item row="0" column="3">
           <widget class="QLCDNumber" name="lcd1">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QLCDNumber" name="lcd2"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="l2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ряды</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="l1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Мины</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="s3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background:rgb(155, 191, 215);
background: rgb(155, 191, 215);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: ;
	color: rgb(155, 191, 215);
border-color: rgb(155, 191, 215);
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="l3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Колонки</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSlider" name="s2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background:rgb(155, 191, 215);
background: rgb(155, 191, 215);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: ;
	color: rgb(155, 191, 215);
border-color: rgb(155, 191, 215);
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSlider" name="s1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background:rgb(155, 191, 215);
background: rgb(155, 191, 215);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: ;
	color: rgb(155, 191, 215);
border-color: rgb(155, 191, 215);
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="apply">
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(155, 191, 215);
  color: rgb(255, 255, 255);
  font-weight: 900;
  border-radius: 8px;
  border: 2px solid rgb(137, 169, 190);
  padding: 5px 15px;
  margin-top: 10px;
}
QPushButton:hover {
  background-color: rgb(137, 169, 190);
  border: 3px solid rgb(155, 191, 215);
}</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GoBack">
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(155, 191, 215);
  color: rgb(255, 255, 255);
  font-weight: 900;
  border-radius: 8px;
  border: 2px solid rgb(137, 169, 190);
  padding: 5px 15px;
  margin-top: 10px;
}
QPushButton:hover {
  background-color: rgb(137, 169, 190);
  border: 3px solid rgb(155, 191, 215);
}</string>
       </property>
       <property name="text">
        <string>Играть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
